Java/JDI/jdi-uitest-web/src/main/java/com/epam/jdi/uitests/web/selenium/elements/complex/DropList.java

```diff
@@ -52,6 +59,41 @@ public DropList(By valueLocator, By optionsNamesLocator, By allOptionsNamesLocat
         this.button = new GetElementType(valueLocator, this);
     }
 
+    public static void setUp(BaseElement el, Field field) {
+        if (!fieldHasAnnotation(field, JDropList.class, IDropList.class))
+            return;
```

Comment:
Please surround with curly braces. And try to avoid if-else (and similar code structures) statements without using curly.